rfcs/text/0000-application-monitoring-for-ga.md

```diff
@@ -0,0 +1,146 @@
+| Centralized configuration for health checks, custom metrics, alerts, and dashboards | white+black-box | [dcp-monitoring](https://github.com/HumanCellAtlas/dcp-monitoring) | [github+terraform](https://github.com/HumanCellAtlas/dcp-monitoring) | adopt as best practice |
+| Metrics for AWS applications and managed services | white+black-box | [CloudWatch Metrics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_concepts.html#Metric) | [console](https://console.aws.amazon.com/cloudwatch/home?region=us-east-1#metricsV2:) | adopt as best practice for AWS |
+| Metrics for GCP applications and managed services | white+black-box | [Stackdriver Metrics](https://cloud.google.com/monitoring/api/metrics) | [console](https://console.cloud.google.com/monitoring) | adopt as best practice for GCP |
+| Metrics based on operational log indices | white+black-box | [Elasticsearch Metrics](http://docs.grafana.org/features/datasources/elasticsearch/) | [dev/integration/staging](https://logs.dev.data.humancellatlas.org/), [prod](https://logs.data.humancellatlas.org/) | use not recommended, but sometimes necessary |
+| Visualization layer that can plot metrics from any datasource | dashboard | [Grafana](https://grafana.com/) | [dev/integration/staging](https://metrics.dev.data.humancellatlas.org/), [prod](https://metrics.data.humancellatlas.org/) | adopt as best practice |
+| Alerting based on CloudWatch Metrics | alerts | [CloudWatch Alarms](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_concepts.html#CloudWatchAlarms) | [github+terraform](https://github.com/HumanCellAtlas/dcp-monitoring) | adopt as best practice |
+| Alert visibility in slack | alerts | [humancellatlas/logs](https://github.com/HumanCellAtlas/logs/tree/master/apps/cwl_to_slack) (slack notifier) | [dev/integration/staging](https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/logs/pipelines), [prod](https://allspark-prod.data.humancellatlas.org/HumanCellAtlas/logs) | adopt as best practice |
+| Build statuses, system health, and availability reporting | black-box | [Status API](https://github.com/HumanCellAtlas/status-api) | [dev/integration/staging](https://status.dev.data.humancellatlas.org/), [prod](https://status.data.humancellatlas.org/) | adopt as best practice |
+| Public presentation of build statuses, system health, and availability reporting | black-box | [DCP Status Page](https://github.com/HumanCellAtlas/humancellatlas.github.io) | [https://humancellatlas.github.io/](https://humancellatlas.github.io/) | temporary until formal status page is built |
+
+### Black-box monitoring
+
+Black-box monitoring attempts to test system behavior as a user would see it.
```

Comment:
Does this cover only external users (data consumers and possibly submitters) or also internal users, such as ingests use of the datastore?

Reply:
Everyone. Teams need to know if their system is down.

Reply:
Could you make this explicit?